### PR TITLE
All types docs

### PR DESCRIPTION
**Pull request by asmacdo**
https://pulp.plan.io/issues/2561

### CODE REVIEW

File: docs/user-docs/getting_started.rst [hunk 1]
```diff
@@ -141,15 +148,30 @@ specify the feed URL as well as a comma separated list of package names you wish
...
+   Publishing Python Content.
+   [-]
+   ... completed
+   
+   Making files available via web.
+   [-]
+   ... completed
+   
+   
+   Task Succeeded
```
Comment (dkliban) [inline docs/user-docs/getting_started.rst]: Can you please add a section on how to sync from one pulp python repository to another pulp python repository. Thanks!

**Comment by dkliban** [comment]
ok test

**Comment by dkliban** [comment]
ok test

**Comment by dkliban** [comment]
ok test

**Comment by dkliban** [comment]
ok test

**Comment by dkliban** [comment]
ok test

**Comment by dkliban** [comment]
ok test

**Comment by dkliban** [comment]
ok test

**Comment by dkliban** [comment]
ok test

**Comment by dkliban** [comment]
ok test

File: docs/user-docs/getting_started.rst (hunk 1 above)
Comment (asmacdo) [inline docs/user-docs/getting_started.rst]: Added in most recent push.

File: docs/user-docs/getting_started.rst [hunk 2]
```diff
@@ -141,15 +146,42 @@ specify the feed URL as well as a comma separated list of package names you wish
...
+   Making files available via web.
+   [-]
    ... completed
 
 
-   Task Succeeded 
+   Task Succeeded
+
+
+Synchronize Packages from another Pulp
```
Comment (dkliban) [inline docs/user-docs/getting_started.rst]: s/Packages/Python Packages/

File: docs/user-docs/getting_started.rst [hunk 3]
```diff
@@ -141,15 +146,42 @@ specify the feed URL as well as a comma separated list of package names you wish
...
+--------------------------------------
+
+Version 2.0+ publishes include metadata for each project. This allows separate instances of Pulp
+to sync from each other. After a publish is complete, the web publish location can be
+used as a feed for a second Pulp. Sync as normal.::
```
Comment (dkliban) [inline docs/user-docs/getting_started.rst]: s/Sync as normal/Here is an example/

**Comment by dkliban** [comment]
ok test

**Pull request merged**